client/src/components/navbar/NavBar.js

```diff
@@ -0,0 +1,18 @@
+import './NavBar.css';
+import TGIFIcon from '../../assets/TGIF.svg'
```

Comment:
TGIFIcon is currently not being used and the text is being displayed as a link. Is there a reason for this?